### PR TITLE
fix: update default vm size for abe2es

### PR DESCRIPTION
**Pull request by cameronmeissner**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:

```
none
```

### CODE REVIEW

**Review by lilypan26 — APPROVED** [review]

File: e2e/vmss.go [hunk 1]
```diff
@@ -231,7 +231,7 @@ func getBaseVMSSModel(name, sshPublicKey, customData, cseCmd string, opts *scena
 	return armcompute.VirtualMachineScaleSet{
 		Location: to.Ptr(config.Location),
 		SKU: &armcompute.SKU{
-			Name:     to.Ptr("standard_d2s_v4"),
+			Name:     to.Ptr("Standard_DS2_v2"),
```
Comment (Devinwong) [inline e2e/vmss.go:234]: do we not need to change the line `defaultAgentPoolVMSize     = "standard_d2s_v4"` in cluster.go?

Comment (cameronmeissner) [reply e2e/vmss.go:234]: that dictates the vm size for the system agentpool  from which we run the jumpbox pods - those are only created when new clusters are provisioned, thus I don't think we need to

Comment (cameronmeissner) [reply e2e/vmss.go:234]: if it's still an issue we can always change in another follow-up

Comment (djsly) [reply e2e/vmss.go:234]: I think it would be cleaner and lest confusion to centralized the SKU to that constant.

**Pull request merged**